Some System Requirements

devawa opened the pull request:
Updated all that I could need someone to look over it and validate that it makes sense.

devawa committed:
Some System Requirements

Updated all that I could need someone to look over it and validate that it makes sense.

jan-justin reviewed (approved):
I had a quick look through the sections and everything appears to be in order. However, I am uncertain about the second last item under Performance Requirements (Pause and Continue). Perhaps you could elaborate on it?

devawa commented:
Alright will take a look at it and make some adjustments.

devawa committed:
Update Architectural Design Specifications.tex

I elaborated a little bit more on it. If you believe that it makes sense you can merge it with the main branch.
